### ISSUE TITLE
Enhancement: Migrate heavy libraries to peer dependency

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1283]
The proposed libraries that will be moving to peer dependency in an upcoming minor version are:

- puppeteer (replaced with puppeteer-core)
- sharp
- ffmpeg-fluent

**Comment by Urbaee** [comment]
> The proposed libraries that will be moving to peer dependency in an upcoming minor version are:
> 
> - puppeteer (replaced with puppeteer-core)
> - sharp
> - ffmpeg-fluent
Is it no way to send sticker gif in 30fps?

**Comment by smashah** [comment]
@Urbaee you can change the fps using the [processOptions](https://docs.openwa.dev/globals.html#mp4stickerconversionprocessoptions)

https://docs.openwa.dev/classes/client.html#sendmp4assticker

**Comment by Urbaee** [comment]
> @Urbaee you can change the fps using the [processOptions](https://docs.openwa.dev/globals.html#mp4stickerconversionprocessoptions)
> 
> https://docs.openwa.dev/classes/client.html#sendmp4assticker

Yeah i have try that
But when i try 30fps and duration of video is 5 second, it give me an error 

```Video must less than 1MB```

**Comment by smashah** [comment]
@Urbaee yes that's a limit of sticker file sizes from WA. Which is why default fps is 10 - https://docs.openwa.dev/globals.html#defaultprocessoptions

**Comment by Urbaee** [comment]
> @Urbaee yes that's a limit of sticker file sizes from WA. Which is why default fps is 10 - https://docs.openwa.dev/globals.html#defaultprocessoptions

sorry, is there have a function ```client.sendStickerfromUrl``` ?

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.4.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.3.1...3.4.0) (2021-02-08)

- Release 3.4.0 [`55d321e`](https://github.com/open-wa/wa-automate-nodejs/commit/55d321e819023be365ff92288d712f499da8c760)
- BREAKING: ⚠️⚠️⚠️ `ffmpeg-fluent` & `sharp` are now peer dependencies!! #1283 [`bf2240c`](https://github.com/open-wa/wa-automate-nodejs/commit/bf2240c4df438a8342bdfa09ca2d10fc9cc60d60)
- better error handling and preprocessing for stickers [`cbb2f0b`](https://github.com/open-wa/wa-automate-nodejs/commit/cbb2f0b381c71f444b98859cee476fafda0db079)
- fix: `eventMode : true` now work as expected for licensed listeners [`6e47ac1`](https://github.com/open-wa/wa-automate-nodejs/commit/6e47ac114d215d46e9894a73f7704a86103f7b33)
- local `webp` processing error handling improvement [`9a77d5c`](https://github.com/open-wa/wa-automate-nodejs/commit/9a77d5ce7fbda6007e0702285f59b22c2f7e75ed)
- fix `removebg` docs [`3906f6d`](https://github.com/open-wa/wa-automate-nodejs/commit/3906f6dc6c3dfed766f0e706ce6ae4a796f65b92)
- fix: `eventMode` sets up properly in initializer [`04af470`](https://github.com/open-wa/wa-automate-nodejs/commit/04af470eb93a9f361914ccce493de8db82c5a0a7)

**Comment by smashah** [comment]
Please install `sharp` and `ffmpeg-fluent` manually to continue using local sticker processing:

```bash
> npm i -g sharp ffmpeg-fluent
```
